esp/services/ws_workunits/ws_workunitsQuerySets.cpp

```diff
@@ -1158,7 +1158,7 @@ void retrieveQuerysetDetails(IEspContext &context, IArrayOf<IEspWUQuerySetDetail
     retrieveQuerysetDetails(context, details, registry, type, value, cluster, queriesOnCluster);
 }
 
-IPropertyTree* getQueriesOnCluster(const char *target, const char *queryset, bool checkAllNodes)
+IPropertyTree* getQueriesOnCluster(const char *target, const char *queryset, StringArray &queryIDs, bool checkAllNodes)
```

Comment:
Up to you, but if you make this "StringArray *queryIDs" then where you define emptyIDs below you could just use nullptr instead... and when you define queryIDs bellow you can pass by address "&queryIDs".